perfkitbenchmarker/linux_packages/redis_server.py

```diff
@@ -15,11 +15,25 @@
 
 """Module containing redis installation and cleanup functions."""
 
+from perfkitbenchmarker import flags
 from perfkitbenchmarker import vm_util
 
-REDIS_TAR = 'redis-2.8.9.tar.gz'
-REDIS_DIR = '%s/redis-2.8.9' % vm_util.VM_TMP_DIR
+
+flags.DEFINE_integer('redis_total_num_processes', 1,
+                     'Total number of redis server processes.')
+
+flags.RegisterValidator(
```

Comment:
thanks. I was trying to do more complicated validation before due to redis cluster feature. But it is not necessary anymore.